### DISCUSSION TITLE
How can I auto-submit a Conform form when the form becomes valid without requiring a button click?

### DISCUSSION THREAD

**Discussion by pascal-codetaal** [edmundhung/conform#762] [Q&A]
Hello, I'm using Conform for form validation in my React app, and it works great for both client-side and server-side validation. However, I would like to enhance the user experience by auto-submitting the form as soon as it becomes valid, without requiring the user to manually click a submit button.

Specifically, I would like to:

Validate the form on every input change.
Auto-submit the form as soon as the Conform validation passes all rules (both native HTML5 + custom validation).
Implement this in Remix .

**Comment by pascal-codetaal** [comment]
This isn't possible?

**Reply by Pouet--** [reply]
Did you try to use a useEffect() who detects form.valid changes, and useSubmit inside, in the case the form is valid ?
